fastlane/lanes/build.rb

```diff
@@ -334,7 +358,8 @@ def generate_prototype_build_number
   # Builds a Prototype Build for WordPress or Jetpack, then uploads it to App Center and comment with a link to it on the PR.
   #
   # rubocop:disable Metrics/AbcSize
-  def build_and_upload_prototype_build(scheme:, output_app_name:, appcenter_app_name:, app_icon:, sentry_project_slug:)
+  # rubocop:disable Metrics/ParameterLists
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/3f5b7cb46a4929942fc8a45bb49d38cf7dc7f8dd/fastlane/lanes/build.rb#L361"></span>Lint/MissingCopEnableDirective: Re-enable Metrics/ParameterLists cop with <code># rubocop:enable</code> after disabling it.</td>
    </tr>
  </tbody>
</table>